### ISSUE TITLE
The result of `graphics/{index}_gl_support_{product_slug}` is hard to read

### ISSUE THREAD

**Issue by seankingyang** [canonical/checkbox#629]
### Bug Description

`graphics/{index}_gl_support_{product_slug}` job result in io-log will be:
```
OpenGL vendor string:   Intel
OpenGL renderer string: Mesa Intel(R) Xe Graphics (TGL GT2)
OpenGL version string:  4.6 (Compatibility Profile) Mesa 22.2.5-0ubuntu0.1~22.04.3

Not software rendered:    ^[[32;01myes^[[00m
Not blacklisted:          ^[[32;01myes^[[00m
GLX fbconfig:             ^[[32;01myes^[[00m
GLX texture from pixmap:  ^[[32;01myes^[[00m
GL npot or rect textures: ^[[32;01myes^[[00m
GL vertex program:        ^[[32;01myes^[[00m
GL fragment program:      ^[[32;01myes^[[00m
GL vertex buffer object:  ^[[32;01myes^[[00m 
GL framebuffer object:    ^[[32;01myes^[[00m
GL version is 1.4+:       ^[[32;01myes^[[00m

Unity 3D supported:       ^[[32;01myes^[[00m

```
The `yes` or `no` will be bound with the color string, it can be easy show this in terminal but hard to read in text only

### To Reproduce

`usr/lib/nux/unity_support_test -p 2>&1 | tee /path/to/file`
read `/path/to/file` in text.

### Environment

No relate with env

**Comment by pieqq** [comment]
Several paths to explore:

- is there an option in the tool to not print color codes?
- is there a flag or an env variable that can be set to let the command know it will print to a device without color code support, and therefore these codes should be stripped?

**Comment by stanley31huang** [comment]
I think we could simply trim the ansi code by sed likes
```
stanley@stanley-ThinkPad-T490:~/Desktop/Git_Repos/iot/checkbox-ce-oem$ /usr/lib/nux/unity_support_test -p 2>&1 | sed -e 's/\x1b\[[0-9;]*m//g' | tee unity.log
OpenGL vendor string:   Intel
OpenGL renderer string: Mesa Intel(R) UHD Graphics (CML GT2)
OpenGL version string:  4.6 (Compatibility Profile) Mesa 21.2.6

Not software rendered:    yes
Not blacklisted:          yes
GLX fbconfig:             yes
GLX texture from pixmap:  yes
GL npot or rect textures: yes
GL vertex program:        yes
GL fragment program:      yes
GL vertex buffer object:  yes
GL framebuffer object:    yes
GL version is 1.4+:       yes

Unity 3D supported:       yes
```

**Comment by zongminl** [comment]
Close as fixed by #1031
